src/pages/patientView/clinicalInformation/ClinicalInformationPatientTable.tsx

```diff
@@ -33,14 +34,14 @@ export default class ClinicalInformationPatientTable extends React.Component<
         value: string;
     }): string {
         let ret: string;
-        switch (data.attribute) {
-            case 'Overall Survival (Months)':
-                ret = parseInt(data.value, 10).toFixed(0);
-                break;
-            default:
-                ret = data.value;
-                break;
+        const parsedFloat = parseFloat(data.value);
```

Comment:
data.value is often not numberical, so shouldn't be treated this way.  You need to make sure it's numeric.
![image](https://user-images.githubusercontent.com/186521/222236621-4f75166e-7a96-4152-a40a-73828ff80bf6.png)

Reply:
Corrected this with https://github.com/cBioPortal/cbioportal-frontend/pull/4531/commits/fa9cc113a0d14393cce9f54f25074fde23cbf511